python/GafferTest/MetadataAlgoTest.py

```diff
@@ -465,6 +465,67 @@ def testNumericBookmarks( self ) :
 		self.assertEqual( Gaffer.MetadataAlgo.getNumericBookmark( s, 1 ), None )
 		self.assertEqual( Gaffer.MetadataAlgo.numericBookmark( s["n2"] ), 0 )
 
+	def testNumericBookmarksSerialisation( self ) :
+
+		s = Gaffer.ScriptNode()
+		s["n1"] = Gaffer.Node()
+
+		Gaffer.MetadataAlgo.setNumericBookmark( s, 1, s["n1"] )
```

Comment:
Let's also test a bookmark other than `1` in here, so we have some coverage for the bug noted above.